features/netsocket/LoRaRadio.h

```diff
@@ -0,0 +1,399 @@
+/**
```

Comment:
Well, yes it could be moved. But as a counter argument, I would say that this class is independent of any LoRaWAN code. This is a template for the radio drivers to be written for Mbed-OS in general. If you are adamant , I can move this, otherwise it makes sense to me to keep it where it is. WifiInterface lives there, so does CellularBase, and even LoRaWANBase. I am not sure if netsocket, as its name suggests is used only for network sockets. I think in reality, it's being used for providing any network interfaces there may be.

Reply:
This file should be alongside other lora code, not netsocket. if you need lora/<thisfile> and lora/lorawan/<everything else> that is fine.